fiftyone/brain/internal/core/uniqueness.py

```diff
@@ -124,8 +128,13 @@ def compute_uniqueness(
         progress=progress,
     )
 
+    similarity_method = config.backend.build()
+    similarity_index = similarity_method.initialize(
+        samples=samples, brain_key="", embeddings=embeddings
```

Comment:
https://github.com/voxel51/fiftyone-brain/pull/201/files#diff-ff031633e3c6fa5c934a33c1d37ef31e987ef15d53d7027996b9235bd92e0e98R103

Do I need to set `brain_key` corresponding to the similarity backend here? It is set to `uniqueness_field` in L103.